### PR TITLE
Test Center Done. BackendTest Done

### PR DESCRIPTION
**Pull request by FaridHai**
Frontend changes will be changed again in next Pull Request. Can be ignored here.

### CODE REVIEW

File: backend/src/test/java/de/haiderzada/backend/controller/IngredientsItemControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    @Test
+    public void getfilteredRecipeListfromDB() {
+        //GIVEN
+        Ingredient ingredient = new Ingredient("9999","Banane");
+        Ingredient ingredient1 = new Ingredient("9998","Brot");
+        Ingredient ingredient2 = new Ingredient("9997","Nutella");
+
+        repository.save(ingredient);
+        repository.save(ingredient1);
+        repository.save(ingredient2);
```
Comment (christophersiem) [inline backend/src/test/java/de/haiderzada/backend/controller/IngredientsItemControllerTest.java:68]: I'd save them just like in the test before for consistency

```suggestion
        repository.save(new Ingredient("9999","Banane"));
        repository.save(new Ingredient("9998","Brot"));
        repository.save(new Ingredient("9997","Nutella");
```

File: frontend/src/components/AddIngredients.js [hunk 2]
```diff
@@ -47,22 +49,36 @@ export default function AddIngredients({
...
     background-color: #008080;
     color: white;
-    border-radius: 12px;
   }
-`;
+  .addbutton {
+   margin-top: 24px;
+   border-radius: 12px;
+   min-width: 8px;
+   padding: 8px !important;
```
Comment (christophersiem) [inline frontend/src/components/AddIngredients.js:82]: `!important`is not good practice. You should consider to look for another solution :)

File: frontend/src/components/Search.js [hunk 3]
```diff
@@ -33,5 +33,6 @@ const Wrapper = styled.div`
   grid-template-rows: min-content min-content min-content min-content 1fr;
   Button {
     background-color: #008080;
+    border-radius: 0px;
```
Comment (christophersiem) [inline frontend/src/components/Search.js:36]: do you need to set the border-radius here?

**Review by christophersiem — APPROVED** [review]
lgtm, some comments :)

File: frontend/src/components/Search.js (hunk 3 above)
Comment (FaridHai) [reply frontend/src/components/Search.js:36]: Yes. I have to overwrite the standard Border-radius from Material UI.

File: frontend/src/components/AddIngredients.js (hunk 2 above)
Comment (FaridHai) [reply frontend/src/components/AddIngredients.js:82]: Didn't found another solution ^^

File: backend/src/test/java/de/haiderzada/backend/controller/IngredientsItemControllerTest.java (hunk 1 above)
Comment (FaridHai) [reply backend/src/test/java/de/haiderzada/backend/controller/IngredientsItemControllerTest.java:68]: When I save them directly, I can't use them in the next line again.

**Pull request merged**